src/board.py

```diff
@@ -11,7 +11,7 @@
 """
 
 
-__version__ = "0.0.0+auto.0"
+__version__ = "8.12.0"
```

Comment:
I think these versions are supposed to stay on the 0s / auto. It gets filled in by something further down the chain I believe.

Reply:
Yup!  The CI finds that string and replaces it so it can be inside the wheels/scource distributions.  It gets replaced with whatever the GitHub Release tag is.